fresque/default_config.py

```diff
@@ -21,7 +21,7 @@
 # secret key used to generate unique csrf token
 SECRET_KEY = 'Change-me-Im-famous'
 
-## If the authentication method is `fas`:
+# If the authentication method is `fas`:
 # To log in, the user must be a member of one of these groups
 REQUIRED_GROUPS = ('packager', 'provenpackager')
```

Comment:
Well, it's not about  new group, it's about the status of someone in that group (user, sponsor, administrator)